bothan-core/src/registry/post_processor.rs

```diff
@@ -22,7 +23,7 @@ pub trait PostProcess<T> {
 }
 
 /// The PostProcess enum represents the different types of post-processors that can be used.
-#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize, Encode, Decode)]
```

Comment:
if we add more enum variants will it be backward-compatible?

Reply:
yes, bincode encodes enum as u32 so adding more shouldn't be an issue